packages/react-router/index.ts

```diff
@@ -353,6 +354,118 @@ export {
   type MetaMatch,
 } from "./lib/router/define-route";
 
+// Expose old @remix-run/server-runtime API, minus duplicate APIs
+export { createCookieFactory, isCookie } from "./lib/server-runtime/cookies";
+export {
+  composeUploadHandlers as unstable_composeUploadHandlers,
+  parseMultipartFormData as unstable_parseMultipartFormData,
+} from "./lib/server-runtime/formData";
+// TODO: (v7) Clean up code paths for these exports
+// export {
+//   defer,
+//   json,
+//   redirect,
+//   redirectDocument,
```

Comment:
@brophdawg11 What's the plan with these APIs? For now I've left the React Router versions as the exported implementations.